pkgs/top-level/all-packages.nix

```
@@ -38526,7 +38526,9 @@ with pkgs;
 
   eiciel = callPackage ../tools/filesystems/eiciel { };
 
-  dpkg = callPackage ../tools/package-management/dpkg { };
+  dpkg = callPackage ../tools/package-management/dpkg {
+    inherit (darwin.apple_sdk.frameworks) CoreServices;
+  };
```

```suggestion
  dpkg = callPackage ../tools/package-management/dpkg { };
```
that is now kinda an anti pattern and we want to stop using it.

---

ok, respun